examples/TheoJansenStrandbeest.cpp

```diff
@@ -415,7 +415,7 @@ void UserInputHandler::handleEvent(State& state, Real accuracy,
     }  
 }
 
-
+/*
```

Comment:
Line 428, comment which is ending with "\" is generating warning. Quick fix for clarity or it could be ignored.